sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -127,7 +130,7 @@
  *
  * <h3>Reading</h3>
  *
- * <p>To read from a BigQuery table, apply a {@link BigQueryIO.Read} transformation. This produces a
+ * <p>To read from a BigQuery table, apply a {@link Read} transformation. This produces a
```

Comment:
The class-level comment should mention the new methods and ideally have a short example.